### PR TITLE
CUMULUS-436: Add activities support to @cumulus/integration-tests

### PR DESCRIPTION
**Pull request by abarciauskas-bgse**
**Summary:** Adds support for testing output of activities in workflow tests

Addresses [CUMULUS-436: Cumulus Integration Tests should include an ECS task](https://bugs.earthdata.nasa.gov/browse/CUMULUS-436)

## Changes

* Replaces `integration-tests/lambda.js` with `integration-tests/sfnStep.js` and removes many reference to `lambda`
* `sfnStep.js` includes `LambdaStep` and `ActivityStep` classes which inherit from `SfnStep`
* Update CHANGELOG
* Update exports from `integration-tests/index.js`

## Test Plan

- [x] N/A Unit tests
- [x] Integration tests passing (see https://github.com/cumulus-nasa/cumulus-integration-tests/pull/10)
- [x] Updated CHANGELOG

### CODE REVIEW

File: packages/integration-tests/sfnStep.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    if (stepExecution.completeEvent === null ||
+        stepExecution.completeEvent.type !== this.successEvent) {
+      console.log(`Step ${stepName} was not successful.`);
+      return null;
+    }
+
+    const succeededDetails = JSON.parse(stepExecution.completeEvent[this.eventDetailsKeys.succeeded].output.toString());
+    return succeededDetails;
```
Comment (laurenfrederick) [inline packages/integration-tests/sfnStep.js]: nitpicky, but you don't need to define `succeededDetails`, you could just return the value.

**Comment by sethvincent** [comment]
This changes the API for getting the lambda output from `getLambdaOutput` to `lambdaStep.getStepOutput` so should technically be a major version bump and I'd put that change under `Changed` in addition to what's under `Added`.

File: packages/integration-tests/index.js [hunk 2]
```diff
@@ -69,11 +69,13 @@ async function waitForCompletedExecution(executionArn) {
   let statusCheckCount = 0;
 
   // While execution is running, check status on a time interval
+  /* eslint-disable no-await-in-loop */
```
Comment (yjpa7145) [inline packages/integration-tests/index.js]: nit: I usually prefer to add the eslint-disable comment on the same line that is causing the error.  So, something like this:

`await timeout(waitPeriodMs); // eslint-disable-line no-await-in-loop`

File: packages/integration-tests/index.js [hunk 3]
```diff
@@ -123,7 +125,7 @@ async function startWorkflowExecution(workflowArn, inputFile) {
 async function executeWorkflow(stackName, bucketName, workflowName, inputFile) {
   const workflowArn = await getWorkflowArn(stackName, bucketName, workflowName);
   const execution = await startWorkflowExecution(workflowArn, inputFile);
-  const executionArn = execution.executionArn;
+  const { executionArn } = execution;
```
Comment (yjpa7145) [inline packages/integration-tests/index.js]: These two lines could just be

`const { executionArn } = await startWorkflowExecution(workflowArn, inputFile);`

File: packages/integration-tests/sfnStep.js [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+    // Get the event where the step was scheduled
+    const scheduleEvent = executionHistory.events.find((event) => {
+      const eventScheduled = this.scheduleEvents.includes(event.type);
+      const eventDetails = event[this.eventDetailsKeys.scheduled];
+      const isStepEvent = eventDetails && eventDetails.resource.includes(stepName);
+      return eventScheduled && isStepEvent;
+    });
+
+    if (scheduleEvent === null || scheduleEvent === undefined) {
```
Comment (yjpa7145) [inline packages/integration-tests/sfnStep.js]: This could just be `if (!scheduleEvent) {`

**Comment by abarciauskas-bgse** [comment]
Thanks @laurenfrederick @yjpa7145 for your reviews! Addressed comments and just waiting for CI to pass before merging.

**Pull request merged**